packages/authorization-platformstate-writer/Dockerfile

```diff
@@ -3,7 +3,7 @@ ARG NODE_REGISTRY="docker.io"
 
 FROM ${NODE_REGISTRY}/node:20.14.0-slim@sha256:5e8ac65a0231d76a388683d07ca36a9769ab019a85d85169fe28e206f7a3208e as build
 
-RUN corepack enable && corepack prepare pnpm@9.15.4 --activate
+RUN npm install -g corepack@0.31.0 && corepack enable
```

Comment:
I think that if we update node to the latest 20 version (`20.19.3`) in the line above, we can even remove this line, since node 20 already installs corepack 0.32.0, which is even newer than this version. If you do that and test that it works, update also `@types/node` in all the package.json files

```suggestion
```

Reply:
True, `RUN corepack enable` is still needed, the fix that was applied (and now is not needed anymore) started from there: https://github.com/pagopa/interop-be-monorepo/pull/1444/files

So we can leave `RUN corepack enable`